src/styled/shared-card.ts

```diff
@@ -0,0 +1,90 @@
+import styled from '@emotion/styled';
+import theme from '@mdb/flora/theme';
+import { HorizontalRule, Pill, TypographyScale } from '@mdb/flora';
+import { PillCategory } from '../types/pill-category';
+import { pillColorMap } from './pill-color';
+import Image from 'next/image';
+
+const SharedCardWrapper = styled('div')`
+    padding: 32px;
+    height: max-content;
+    border: 1px solid #e7eeec;
+    box-shadow: 0px 2px 4px rgba(0, 0, 0, 0.1);
+    border-radius: ${theme.radii.inc50};
+    // Mobile
+    width: 467px;
+    // Desktop small
+    @media only screen and (min-width: ${theme.sizes.breakpoint.large}) {
+        width: 336px;
+    }
+`;
+
+const ThumbnailImage = styled(Image)`
+    border-radius: 8px;
+`;
+
+const ThumbnailWrapper = styled('div')`
+    // Mobile
+    margin-bottom: ${theme.space.elementXSmall};
+    // Desktop small
+    @media only screen and (min-width: ${theme.sizes.breakpoint.large}) {
+        margin-bottom: ${theme.space.elementSmall};
+    }
+`;
+
+const IntrinsicRatioWrapper = styled('div')`
+    position: relative;
+    width: 100%;
```

Comment:
This is a block element so you don't need to declare it to take 100% of the width.